packages/eslint-plugin/src/rules/keyword-spacing.ts

```diff
@@ -0,0 +1,293 @@
+import baseRule from 'eslint/lib/rules/keyword-spacing';
+import ESLINT_UTILS_KEYWORDS from 'eslint/lib/rules/utils/keywords';
+import {
+  TSESTree,
+  AST_TOKEN_TYPES,
+} from '@typescript-eslint/experimental-utils';
+import { isTokenOnSameLine } from '../util/astUtils';
+import * as util from '../util';
+
+export type Option = Partial<{
+  before: boolean;
+  after: boolean;
+}>;
+export type OverrideOptions = Partial<Record<string, Option>>;
+export type RootOption = Option & { overrides?: OverrideOptions };
+
+export type Options = util.InferOptionsTypeFromRule<typeof baseRule>;
+export type MessageIds = util.InferMessageIdsTypeFromRule<typeof baseRule>;
+
+const PREV_TOKEN = /^[)\]}>]$/u;
+const NEXT_TOKEN = /^(?:[([{<~!]|\+\+?|--?)$/u;
+const TEMPLATE_OPEN_PAREN = '${';
+const TEMPLATE_CLOSE_PAREN = '}';
+const CHECK_TYPE = /^(?:JSXElement|RegularExpression|String|Template)$/u;
+
+const KEYWORDS = ESLINT_UTILS_KEYWORDS.concat([
+  // Eslint Utils does no provide all keywords, so we complete the list here.
+  'as',
+  'async',
+  'await',
+  'from',
+  'get',
+  'let',
+  'of',
+  'set',
+  'yield',
+]);
+
+// Check duplications.
+(function(): void {
+  KEYWORDS.sort();
+  for (let i = 1; i < KEYWORDS.length; ++i) {
+    if (KEYWORDS[i] === KEYWORDS[i - 1]) {
+      throw new Error(
+        `Duplication was found in the keyword list: ${KEYWORDS[i]}`,
+      );
+    }
+  }
+})();
```

Comment:
could just use a set

```suggestion
const KEYWORDS = new Set(ESLINT_UTILS_KEYWORDS.concat([
  // Eslint Utils does no provide all keywords, so we complete the list here.
  'as',
  'async',
  'await',
  'from',
  'get',
  'let',
  'of',
  'set',
  'yield',
]));
```